content/2_Bronze/Intro_DS.mdx

```diff
@@ -710,6 +709,118 @@ See [here](/general/notes-cpp-ds).
 
 </LanguageSection>
 
+## Quiz
+
+<LanguageSection>
```

Comment:
Not sure but is the language section necessary for quizzes

Reply:
yes, since the questions are different for different languages